scripts/release.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#
+# Note: The following environment variables are set by container-release build job
+#   - CODEBUILD_INITIATOR
+#   - CODEBUILD_BUILD_NUMBER
+#   - TERRAFORM_PROVISIONER_ROLE_ARN
+
+set -euo pipefail
+IFS=$'\n\t'
+
+ASSUME_ROLE_ARN="${LABS_TERRAFORM_PROVISIONER_ROLE_ARN}"
+
+set_aws_credentials() {
```

Comment:
Worth drying up these scripts? Probably not...

Reply:
You have a really good point. I was hoping to keep the scripts as self-contained as possible. Also, I expect the bash scripts to go through quite a bit of refactoring when the `LIVE` environment is covered. This is going to happen in the next iteration. If it is ok, I'd like to defer the idea until then.